src/database/RecipeEntity.js

```diff
@@ -1,3 +1,84 @@
 const {Datastore} = require("@google-cloud/datastore");
 const datastore = new Datastore();
 
+async function getRecipeEntities(pageCursor) {
+  let query = datastore.createQuery("Recipe");
+
+  if (pageCursor) {
+    query = query.start(pageCursor);
+  }
+
+  const results = await datastore.runQuery(query);
+  const entities = results[0];
+  const info = results[1];
+
+  if (info.moreResults !== Datastore.NO_MORE_RESULTS) {
+    const results = await getRecipes(info.endCursor);
+    return [...entities, results[0]];
+  }
+  return entities;
+}
+
+async function getRecipes(data) {
```

Comment:
nit: data -> filterparams?

Or a comment in the method for what data is supposed to be.